business/services.go

```diff
@@ -75,31 +82,64 @@ func (in *SvcService) GetServiceList(namespace string) (*models.ServiceList, err
 
 	go func() {
 		defer wg.Done()
-		var err error
+		var err2 error
 		// Check if namespace is cached
 		// Namespace access is checked in the upper call
 		if IsNamespaceCached(namespace) {
-			deployments, err = kialiCache.GetDeployments(namespace)
+			deployments, err2 = kialiCache.GetDeployments(namespace)
 		} else {
-			deployments, err = in.k8s.GetDeployments(namespace)
+			deployments, err2 = in.k8s.GetDeployments(namespace)
 		}
-		if err != nil {
-			log.Errorf("Error fetching Deployments per namespace %s: %s", namespace, err)
-			errChan <- err
+		if err2 != nil {
+			log.Errorf("Error fetching Deployments per namespace %s: %s", namespace, err2)
+			errChan <- err2
 		}
 	}()
 
+	linkedResources := map[string]*[]kubernetes.IstioObject{
```

Comment:
Don't you want to apply the same changes in here?

Reply:
Ops I missed, sorry.